docs/usage/configuration-reference.md

```diff
@@ -40,16 +40,17 @@ The path object defines the paths to some of the key directories used by Shunter
 
 ```js
 path: {
-    root: appRoot,
-    shunterRoot: shunterRoot,
-    themes: path.join(shunterRoot, 'themes'),
-    templates: path.join(appRoot, 'view'),
+    clientTests: path.join(appRoot, 'tests', 'client'),
+    dust: path.join(appRoot, 'dust'),
     public: path.join(appRoot, 'public'),
     publicResources: path.join(appRoot, 'public', 'resources'),
     resources: path.join(appRoot, 'resources'),
+    root: appRoot,
     shunterResources: path.join(shunterRoot, 'resources'),
+    shunterRoot: shunterRoot,
+    templates: path.join(appRoot, 'view'),
     tests: path.join(appRoot, 'tests'),
-    dust: path.join(appRoot, 'dust')
+    themes: path.join(shunterRoot, 'themes')
```

Comment:
Is the alphabetical reordering appropriate here?  Maybe things of the same category belonged together.